views/js/review/plugins/navigation/review-panel/panel.js

```diff
@@ -26,9 +26,13 @@ define([
     'ui/autoscroll',
     'ui/component',
     'ltiTestReview/review/plugins/navigation/review-panel/panel-data',
-    'tpl!ltiTestReview/review/plugins/navigation/review-panel/tpl/panel',
-    'tpl!ltiTestReview/review/plugins/navigation/review-panel/tpl/list',
-    'css!ltiTestReview/review/plugins/navigation/review-panel/css/panel.css'
+    'tpl!ltiTestReview/review/plugins/navigation/review-panel/tpl/accordionPanel',
+    'tpl!ltiTestReview/review/plugins/navigation/review-panel/tpl/accordionList',
+    'tpl!ltiTestReview/review/plugins/navigation/review-panel/tpl/fizzyPanel',
+    'tpl!ltiTestReview/review/plugins/navigation/review-panel/tpl/stepOverview',
```

Comment:
The version in #99 with the separated controllers does seem like a more sensible way of handling this, even if it leads to increased duplication.